### ISSUE TITLE
React-Pose 4.0.x bug - Child pose state no effect if parent was assigned pose props

### ISSUE THREAD

**Issue by meepeek** [Popmotion/popmotion#655]
## Ensure you're using the latest version
Tested on 3.4.0, 4.0.0, 4.0.2

## Add the package name to bug title
React-Pose 4.0.x bug - Child pose state not effect if parent was assigned pose props

## Describe the bug
I have 2 pose div, Parent and Child like this:
```
const Child = posed.div({
hide: {opacity: 0}
})
<Parent pose={'anything'}><Child pose={'hide'}>Pose 3.4.0 will not see this but 4.0.x will</Child></Parent>
```
You will see the text in Pose version 4.0.x which is a bug

## How to reproduce
1. Go to the following codepen
2. Change version of pose to 3.4.0 you will not see the text
3. Change version of pose to 4.0.2 you will see the text again

## Expected behaviour
The text should not be seen

## Link to CodePen example (or similar)
https://codepen.io/anon/pen/EOGGMv

## Device information
In which environment are you reproducing the bug?
Chrome Desktop

## Additional context
None

**Comment by Andarist** [comment]
Thanks for the report. I think this commit has introduced the regression - https://github.com/Popmotion/popmotion/commit/1420fada3570e32206e2b82ea625029b9b3fd4f6 . 

Gonna work on the fix and writing a test for this, but I might not have time to do so quickly - probably will do it in next week though.

**Comment by InventingWithMonster** [comment]
Fixed in `react-pose@4.0.3`
